spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/BraveAutoConfiguration.java

```diff
@@ -252,4 +255,59 @@ ScopeDecorator correlationScopeDecorator(CorrelationScopeDecorator.Builder build
 
 	}
 
+	/**
+	 * Factory for {@link Factory propagation factories}.
+	 */
+	private static class PropagationFactoryFactory {
+
+		private PropagationFactoryFactory() {
+		}
+
+		/**
+		 * Creates a new B3 propagation factory.
+		 * @return B3 propagation factory.
+		 */
+		private static Factory b3() {
+			return B3Propagation.newFactoryBuilder().injectFormat(B3Propagation.Format.SINGLE_NO_PARENT).build();
```

Comment:
Just a note for the docs: I would call it out that W3C and B3 single is used by default, if users want B3 multi instead (or both?), they need to create their own `CompositePropagationFactory` instance and `BaggagePropagation.FactoryBuilder` bean.

Reply:
I wonder if it wouldn't be better for compatibility to check for B3 multi but check it last (so it only happens when other, higher precedent formats aren't present). Unless the concern is about the added cost of trying to extract b3 multi when no trace context is present. Perhaps then we could measure the cost. What do you think?

Reply:
For compatibility, it would be definitely better. From cost perspective and trying to move away from B3 multi, it would not. I _assume_ the cost would be minimal since this should be just a map lookup.
If we do this should not we also support it on the output (i.e.: `B3_SINGE` and `B3_MULTI`)?

Reply:
Yeah, probably. I think `B3` for single and `B3_MULTI` make sense.